Sources/Views/OATargetPointView.mm

```diff
@@ -1507,6 +1531,51 @@ - (CGPoint) doLayoutSubviews:(BOOL)adjustOffset
     return newOffset;
 }
 
+- (void) addMapFrameLayer:(CGRect)frame
```

Comment:
It should be introduced in OATargetMenuViewController and customized via OADownloadMapViewController